### PR TITLE
HIVE-23020 : Avoid using _files for replication data copy during incr…

### PR DESCRIPTION
**Pull request by pkumarsinha**
…emental run

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java [hunk 1]
```diff
@@ -505,9 +506,8 @@ private static ImportTableDesc getBaseCreateTableDescFromTable(String dbName,
 
     Task<?> copyTask = null;
     if (replicationSpec.isInReplicationScope()) {
-      boolean isImport = ReplicationSpec.Type.IMPORT.equals(replicationSpec.getReplSpecType());
       copyTask = ReplCopyTask.getLoadCopyTask(replicationSpec, dataPath, destPath, x.getConf(),
-              isAutoPurge, needRecycle, copyToMigratedTxnTable, !isImport);
+              isAutoPurge, needRecycle, copyToMigratedTxnTable, false);
```
Comment (aasha) [inline ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java:510]: Why is this set to false? Do we even need this boolean if everyone is setting it to false?

File: ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AddPartitionHandler.java [hunk 2]
```diff
@@ -108,10 +108,8 @@ public void handle(Context withinContext) throws Exception {
         Iterable<String> files = partitionFilesIter.next().getFiles();
         if (files != null) {
           // encoded filename/checksum of files, write into _files
-          try (BufferedWriter fileListWriter = writer(withinContext, qlPtn)) {
-            for (String file : files) {
-              writeFileEntry(qlMdTable.getDbName(), qlMdTable, file, fileListWriter, withinContext);
-            }
+          for (String file : files) {
```
Comment (aasha) [inline ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AddPartitionHandler.java:111]: tests for this

File: ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java (hunk 1 above)
Comment (pkumarsinha) [reply ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java:510]: Used with true in some cases.

File: ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AddPartitionHandler.java (hunk 2 above)
Comment (pkumarsinha) [reply ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AddPartitionHandler.java:111]: added

**Pull request closed without merging**